packages/sku/lib/packageManager.js

```diff
@@ -51,4 +57,96 @@ const isYarn = packageManager === 'yarn';
 const isPnpm = packageManager === 'pnpm';
 const isNpm = packageManager === 'npm';
 
-module.exports = { rootDir, packageManager, isYarn, isPnpm, isNpm };
+/**
+ * @param {string} scriptName
+ */
+const getRunCommand = (scriptName) =>
+  getCommand(packageManager, 'run', [scriptName]);
+
+/**
+ * @param {string[]} args
+ */
+const getExecuteCommand = (args) => getCommand(packageManager, 'execute', args);
+
+/** @type {Record<SupportedPackageManager, string[]>} */
+const regularLoglevelArgsByPackageManager = {
+  // Yarn doesn't have a loglevel flag
+  yarn: [],
+  pnpm: ['--loglevel', 'error'],
+  npm: ['--loglevel', 'error'],
+};
+
+/** @type {Record<SupportedPackageManager, string[]>} */
+const verboseLoglevelArgsByPackageManager = {
+  yarn: ['--verbose'],
+  pnpm: ['--loglevel', 'info'],
+  npm: ['--loglevel', 'verbose'],
+};
+
+/**
+ * @param {'verbose' | 'regular'} logLevel
+ */
+const resolveLogLevelArgs = (logLevel) => {
+  if (logLevel === 'verbose') {
+    return verboseLoglevelArgsByPackageManager[packageManager];
+  }
+
+  return regularLoglevelArgsByPackageManager[packageManager];
+};
+
+/**
+ * @typedef {object} GetAddCommandOptions
+ * @property {'dev' | 'prod'} type
+ * @property {'verbose' | 'regular' | undefined} logLevel
+ * @property {boolean} exact
+ * @property {string[]} deps
+ * @param {GetAddCommandOptions} options
+ */
+const getAddCommand = ({ type, logLevel, deps, exact }) => {
+  const args = [];
+
+  const addingDevDeps = type === 'dev';
+
+  if (addingDevDeps) {
+    const devDepFlag = isYarn ? '--dev' : `--save-dev`;
+    args.push(devDepFlag);
+  }
+
+  if (exact) {
+    const exactFlag = isYarn ? '--exact' : '--save-exact';
+    args.push(exactFlag);
+  }
+
+  if (logLevel) {
+    args.push(...resolveLogLevelArgs(logLevel));
+  }
+
+  args.push(...deps);
+
+  return getCommand(packageManager, 'add', args);
+};
+
+const getInstallCommand = () => getCommand(packageManager, 'install', []);
+
+const getWhyCommand = () => {
+  const whyCommand = isPnpm ? '-r why' : 'why';
```

Comment:
```
Usage: pnpm [command] [flags]
```

Just to be pedantic...

```suggestion
  const whyCommand = isPnpm ? 'why -r' : 'why';
```

Reply:
https://github.com/seek-oss/sku/pull/884/commits/2a8d5567a740ba4341348b3b7897ab25d859e7d2